src/libs/ajax.js

```diff
@@ -33,7 +34,7 @@ const query = (url, data, method = 'GET') => {
         try {
           jsonResponse = JSON.parse(this.response)
         } catch (e) {
-          console.error('json', this.response)
+          Error.displayOnce('ajax', 'query', 'error parsing json', e)
```

Comment:
Is this error really useful ? Is that not redundant with the rejected error ?

Reply:
Also, some message coudl be improved here. For example line 51 `reject(xhr.status)`. Some more context could be useful (`url`, etc.)

Reply:
you are right, for now i let the caller handle the error.

Reply:
i prefer let the callee handle the error if it need, it already know url etc.